.env.example

```diff
@@ -1,4 +1,5 @@
 DATABASE_URL=
 NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
 CLERK_SECRET_KEY=
+ANTHROPIC_API_KEY=
```

Comment:
I am thinking of open-source APIs to avoid unnecessary costs at this moment. I think Gemini have free or we can setup [GPT-3.5 Turbo](https://platform.openai.com/docs/models#gpt-3-5-turbo)

Reply:
> I am thinking of open-source APIs to avoid unnecessary costs at this moment. I think Gemini have free or we can setup [GPT-3.5 Turbo](https://platform.openai.com/docs/models#gpt-3-5-turbo)

Makes Sense @tyaga001. Should I proceed with setting up the backend for the Gemini API and transitioning out of Claude, or is there a preferred direction you'd like to explore?